File: AlbumLibrary/src/main/java/com/album/ui/annotation/FrescoType.java
package com.album.ui.annotation;

import android.support.annotation.IntDef;

import com.album.AlbumConstant;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

/**
 * by y on 22/08/2017.
 */

@IntDef({AlbumConstant.TYPE_FRESCO_ALBUM, AlbumConstant.TYPE_FRESCO_PREVIEW})
@Retention(RetentionPolicy.SOURCE)
public @interface FrescoType {
}

File: AlbumLibrary/src/main/java/com/album/annotation/FrescoType.kt
package com.album.annotation

import androidx.annotation.IntDef
import com.album.AlbumConstant

/**
 * by y on 22/08/2017.
 */

@IntDef(AlbumConstant.TYPE_FRESCO_ALBUM, AlbumConstant.TYPE_FRESCO_PREVIEW)
@Retention(AnnotationRetention.SOURCE)
annotation class FrescoType
